lib/praxis/types/multipart_array.rb

```diff
@@ -51,9 +51,7 @@ def self.name_type(type=nil)
         @name_type = Attributor.resolve_type type
       end
 
-      def self.payload_type(type=nil, **opts, &block)
-        return @payload_type if type.nil?
-
+      def self.payload_type(type=Attributor::Struct, **opts, &block)
```

Comment:
Yeah, I was wondering if we even needed it to work for that use-case since the instance method of the same name is more useful. But I guess it can't hurt. I will make it so!